script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -332,33 +375,42 @@ I thought I'd be the first to arrive, but several of my classmates were already
 [sha:4f044551b36009743a1d7a0a59b439a8d72fecec]{
 -- Page 65, Offset 16739.
 -- 「おはよ。金曜は体調不良。そっちは朝練終わり？」
-"Morning. I was sick on Friday. Are you done with your morning practice?"
+// Interpreting 終わり as ended / on hold more generally within the context of next line
+"Morning. I was sick on Friday.
```

Comment:
I think this is more like "morning practices over already?"

Reply:
It _does_ seem like that, but then the griping that follows doesn't make as much sense. If both morning and evening practice were _shortened_ for their safety, rather than cancalled, wouldn't you shave the earlier part off the morning practice?

Reply:
pft, I'unno. I'm just pretty dang sure that's what it means.

Reply:
Resolved w/Petrikow